src/sagemaker/session.py

```diff
@@ -1264,6 +1270,24 @@ def __init__(self, s3_data, distribution='FullyReplicated', compression=None,
             self.config['RecordWrapperType'] = record_wrapping
         if input_mode is not None:
             self.config['InputMode'] = input_mode
+        if attribute_names is not None:
+            self.config['DataSource']['S3DataSource']['AttributeNames'] = attribute_names
+        if shuffle_config is not None:
+            self.config['ShuffleConfig'] = {'Seed': shuffle_config.seed}
+
+
+class ShuffleConfig(object):
```

Comment:
The s3 input class uses snake case. Should we do the same here? I'll defer to Lauren on style issues as I'm not sure if there is a style guide they are using.

Reply:
Well, I had a choice of either taking a dict or a class. A dict would work though it forces the user to go look at SageMaker docs to understand that the dict requires a single key called "Seed" with a long value. We also cannot take just the seed as input since SageMaker Training's ShuffleConfig could eventually become a richer object.